frappe/utils/pdf.py

```diff
@@ -196,17 +219,15 @@ def prepare_header_footer(soup):
 				tag.extract()
 
 			toggle_visible_pdf(content)
-			html = frappe.render_template(
-				"templates/print_formats/pdf_header_footer.html",
-				{
-					"head": head,
-					"content": content,
-					"styles": styles,
-					"html_id": html_id,
-					"css": css,
-					"lang": frappe.local.lang,
-					"layout_direction": "rtl" if is_rtl() else "ltr",
-				},
+			mapper = {"header-html": "pdf_header_html", "footer-html": "pdf_footer_html"}
+			hook_func = frappe.get_hooks(mapper.get(html_id))
+			html = frappe.get_attr(hook_func[-1])(
```

Comment:
```suggestion
			id_map = {"header-html": "pdf_header_html", "footer-html": "pdf_footer_html"}
			hook_method = frappe.utils.get_hook_method(id_map.get(html_id))
			html = hook_method(
```

use `get_hook_method` instead?

Reply:
get_hook_method always returns frappe app's method 
![Screenshot 2023-04-17 at 3 58 25 PM](https://user-images.githubusercontent.com/39730881/232459547-6af083a3-c3b8-4166-8059-46126d2d2327.png)

Reply:
seems lika a bug... it should return from last app installed :/